godotproject/enemy-Ant.gd

```diff
@@ -52,8 +52,8 @@ func begin_attack():
 			if RC.name == "Player":
 				attackState = STATE.detected_player_e
 				$ActionDelay.start()
+				$AttackSoundPlayer.play(0)
 	
-	$AttackSoundPlayer.play(0)
```

Comment:
Ok, this seems like it makes sense. When does this play the sound effect? When the ant spots the player, or when the fireball is shot?

Reply:
After the player is detected and a split second before fireball is shot